kafka-rest/src/test/java/io/confluent/kafkarest/controllers/TopicManagerImplTest.java

```diff
@@ -846,6 +851,21 @@ public void deleteTopic_nonExistingCluster_throwsNotFound() throws Exception {
     }
   }
 
+  @Test
+  public void alterTopicPartitions() {
```

Comment:
nit: I quite like the convention of test method naming used in this file, methodNameUnderTest_WhenInputIsThis_ThenOutputIsThis.